services/notifications/pkg/service/service.go

```diff
@@ -86,18 +86,12 @@ func (s eventsNotifier) Run() error {
 	}
 }
 
-func (s eventsNotifier) render(bodyTemplate string, subjTemplate string, values map[string]string) (string, string, error) {
-	msg, err := email.RenderEmailTemplate(bodyTemplate, values, s.emailTemplatePath)
+func (s eventsNotifier) renderL10n(template l10n.NotificationTemplate, values map[string]interface{}) (string, string, error) {
+	subject, _, message, _, err := l10n.ComposeMessage(template, "en_US", s.emailTemplatePath, values)
```

Comment:
You could use the language setting of the user and only fallback to English if the user didn't set a language.

Not exactly sure how to use it though. There is a `GetValue` handler in the settings service and a language setting in the default config of the settings service. Maybe someone can give you a hand with that.

Reply:
setting the profile language in the ui is possible via the account page (see top right "user" menu, then click on the username. There is a dropdown where you can select a language).

Reply:
That is a #4 in an issue description, we can't to determine the users preferred language. That why I set a default locale, but I'm not sure if it is correct. https://github.com/owncloud/ocis/issues/6025

Reply:
> setting the profile language in the ui is possible via the account page (see top right "user" menu, then click on the username. There is a dropdown where you can select a language).

If i understand correctly the locale setting come from a UI. The Userlog service uses the 'Accept-Language' header to determine the location, but the Notifications service doesn't.

Reply:
I the actions in the notifications service always originate from a request, you have the `Accept-Language` header available because ownCloud Web sends it in all requests. That would be good. ownCloud Web sends the `Accept-Language` based on the users profile language with a fallback to the browser language. My assumption was that you don't always have request headers available.

Reply:
@kulmann your assumption is correct. But we are not entirely sure how to easily get the users language. As far as I know the settings endpoint is only used by clients. The server is not aware this setting exists. That's why we split the story in two parts.